Uniswap/interface
# feat: statsig flag for base support

## Description



adds feature flag gating to two functions, `getChainInfo` and `asSupportedChainId`, so that we can selectively block certain chains from appearing in the app.


## Screen capture


https://github.com/Uniswap/interface/assets/66155195/6fc1a586-0774-4d9e-a5a9-23910ca942a5



## Test plan

### QA (ie manual testing)


see video - tested switching chains in both wallet and network-switcher. verified that when disabled we show "connect to ethereum" in the swap page if the wallet is connected to a base network
